src/manifest.json

```diff
@@ -26,27 +20,28 @@
 			"background.js",
 			"lib/500pxwall.zip"
 		]
-	],
+	},
 	"omnibox": {
     "keyword" : "newtab"
 	},
 	"icons": {
-		"128": ": "img/main.png",
+		"128": "img/main.png"
+	},
+	"browser_action": {
+		"default_popup": "action.html",
+		"browser_style": true
```

Comment:
"default_popup" and "browser_style" are not top level keys, but under "browser_action" or "page_action". "browser_action" creates an toolbar icon, and "page_action" creates an icon in the address bar.

Reply:
> "default_popup" and "browser_style" are not top level keys, but under "browser_action" or "page_action". "browser_action" creates an toolbar icon, and "page_action" creates an icon in the address bar.

Noted :+1: